examples/voltage_clamp/voltage_clamp.ipynb

```diff
@@ -0,0 +1,801 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "act_tau_efel = np.array([feat_dict[\"activation_time_constant\"][0] for feat_dict in feats])\n",
+    "plt.plot(stim_v, act_tau_efel, '.')\n",
+    "plt.xlabel(\"Stimulus Voltage (mV)\")\n",
+    "plt.ylabel(\"Activation time constant (ms)\")\n",
+    "plt.xlim((-15, 85))\n",
+    "plt.ylim((0, 10))\n",
+    "plt.title(\"Activation time constant\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deactivation traces\n",
+    "\n",
+    "Now we will also perform efeature extraction on traces from deactivation experiment. First, we have load the data from the file. Once again, we will put the current data under the 'V' key representing the voltage, so that efel treats our trace like any voltage trace. Notice that here, we are setting the stimulus start at 401 and stimulus end at 599 because from 400 to 600 is the interval of interest for our time constant fit, and we start slightly after and end slightly before 'real' stimulus in order to prevent artifacts that show when there is a stimulus change to affect our computation"
```

Comment:
we have load -> we load
change to affect -> change that affect ?

Reply:
Fixed in latest commit